### PR TITLE
refactor(demo): Accordion knobs and labels added - FRONT 909

### PR DESCRIPTION
**Pull request by Joosthe**
# PR description

Please drop a few lines about the PR: what it does, how to test it, etc.

## QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:

- [ ] I have put the vanilla component as `devDependencies`
- [ ] I have put the specs package as `devDependencies`
- [ ] I have added the components directly used in the twig file (with `include` or `embed`) as `dependencies`
- [ ] My component is listed in `@ecl-twig/ec-components`'s `dependencies`
- [ ] My variables naming follow the guidelines (snake case for twig)
- [ ] I have provided tests
- [ ] I have provided documentation (for the "notes" tab)
- [ ] If my local `yarn.lock` contains changes, I have committed it
- [ ] I have given my PR the proper label (`pr: review needed` to indicate that I'm done and now waiting for a review ,`pr: wip` to indicate that I'm actively working on it ...)

### CODE REVIEW

File: src/ec/packages/ec-component-accordion/accordion.story.js [hunk 1]
```diff
@@ -1,36 +1,52 @@
...
 import accordion from './ecl-accordion.html.twig';
 import notes from './README.md';
 
+const preprareAccordion = data => {
+  data.items.forEach((item, index) => {
+    const { id, level, toggle, content } = item;
+    const levels = [1, 2, 3, 4, 5, 6];
+    item.id = select(`item[${index}].id`, [id], id, buttonLabels.required);
+    item.level = select(
+      `item[${index}].level`,
```
Comment (papegaill) [inline src/ec/packages/ec-component-accordion/accordion.story.js]: `items[${index}].level`

File: src/ec/packages/ec-component-accordion/accordion.story.js [hunk 2]
```diff
@@ -1,36 +1,52 @@
...
+      levels,
+      level,
+      buttonLabels.required
+    );
+    item.toggle.label = text(
+      `items[${index}].toggle.label`,
+      toggle.label,
+      buttonLabels.required
+    );
+    item.content = text(
```
Comment (papegaill) [inline src/ec/packages/ec-component-accordion/accordion.story.js]: item.content before item.toggle like accordion2

File: src/ec/packages/ec-component-accordion/accordion.story.js [hunk 3]
```diff
@@ -1,36 +1,52 @@
...
+      `items[${index}].content`,
+      content,
+      buttonLabels.required
+    );
+
```
Comment (papegaill) [inline src/ec/packages/ec-component-accordion/accordion.story.js]: remove empty line

File: src/ec/packages/ec-component-accordion/accordion.story.js [hunk 4]
```diff
@@ -1,36 +1,52 @@
...
+    item.toggle.icon.path = select(
+      `items[${index}].toggle.icon.path`,
+      [defaultSprite],
+      defaultSprite,
+      buttonLabels.required
+    );
+    getExtraKnobs(data);
+  });
+  return data;
```
Comment (papegaill) [inline src/ec/packages/ec-component-accordion/accordion.story.js]: add extra line before return

File: src/ec/packages/ec-component-accordion/accordion.story.js [hunk 5]
```diff
@@ -1,36 +1,52 @@
...
-      });
-
-      return accordion(demoData);
-    },
-    {
-      notes: { markdown: notes, json: demoData },
-    }
-  );
+  .add('ECL < 2.6.0 - default', () => accordion(preprareAccordion(demoData)), {
+    notes: { markdown: notes, json: preprareAccordion(demoData) },
```
Comment (papegaill) [inline src/ec/packages/ec-component-accordion/accordion.story.js]: why are you adding are passing demoData to preprareAccordion function for json notes?

File: src/ec/packages/ec-component-accordion/accordion.story.js (hunk 1 above)
Comment (Joosthe) [reply src/ec/packages/ec-component-accordion/accordion.story.js]: typo updated

File: src/ec/packages/ec-component-accordion/accordion.story.js (hunk 2 above)
Comment (Joosthe) [reply src/ec/packages/ec-component-accordion/accordion.story.js]: like in accordion the Content should not come before the label

File: src/ec/packages/ec-component-accordion/accordion.story.js (hunk 3 above)
Comment (Joosthe) [reply src/ec/packages/ec-component-accordion/accordion.story.js]: line removed

File: src/ec/packages/ec-component-accordion/accordion.story.js (hunk 5 above)
Comment (Joosthe) [reply src/ec/packages/ec-component-accordion/accordion.story.js]: removed the preprareAccordion. function at the notes

File: src/ec/packages/ec-component-accordion/accordion.story.js (hunk 4 above)
Comment (Joosthe) [reply src/ec/packages/ec-component-accordion/accordion.story.js]: extra line added

**Pull request merged**